src/index.js

```diff
@@ -8,6 +8,22 @@ class Azk {
   static get version() {
     return require('package.json').version;
   }
+  static get gitCommitId() {
+    const config = GeralLib.config;
+    const path = GeralLib.path;
+
+    // git commit id from ENV
+    const azk_last_commit = config('azk_last_commit');
+    if (azk_last_commit) {
+      return azk_last_commit;
```

Comment:
Yep, bluebird do this automatically. I will refactor to return a `promiseResolve`.